test/boost/view_schema_test.cc

```diff
@@ -2573,13 +2573,22 @@ SEASTAR_TEST_CASE(test_unselected_column) {
 }
 
 SEASTAR_THREAD_TEST_CASE(node_view_update_backlog) {
-    db::view::node_update_backlog b(2, 10ms);
+    db::view::node_update_backlog b(2, 100ms);
```

Comment:
Nit: why was it necessary to increase the interval?

Reply:
The test seems to be executing slightly slower due to the `invoke_on`s and it was failing with the original interval. It was very rare, less than 1%, but I assume we want the fail to be even less likely than that.